### PR TITLE
Update Copyright Headers

### PR DESCRIPTION
**Pull request by NickGerleman**
This is a quick regex replace. Addresses #3933 to remove "All rights reserved" and #3797 to replace the Facebook license banner.

Some forked JS already had both licences, saying "Portions copyright for react-native-windows:". These were updated to just have the MSFT license, since we mention Facebook in our main LICENSE file.

This is likely to get out of date unless we add CI to catch missing the license banner.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/react-native-windows/pull/4090)

### CODE REVIEW

**Comment by NickGerleman** [comment]
Blegh, VSCode formatted a bunch of JS on replace. Need to fix that.

File: vnext/src/RNTester/js/components/RNTesterExampleList.windows.js [hunk 1]
```diff
@@ -1,8 +1,6 @@
 /**
- * Copyright (c) Facebook, Inc. and its affiliates.
```
Comment (vmoroz) [inline vnext/src/RNTester/js/components/RNTesterExampleList.windows.js]: Same here. We must not remove Facebook copyright notice. Why not just remove the 'All rights reserved' from Microsoft copyright notice?

**Comment by NickGerleman** [comment]
@kmelmon said he looked into this earlier and replacing the headers was the right thing to do. Not sure what context went into that though.

**Comment by NickGerleman** [comment]
> I am a little bit uneasy by us removing the Facebook copyright notice. Could someone please refer to the official LCA open source guidelines before we do that? As far as I remember we must never do it, but my memory may not be correct.

https://docs.opensource.microsoft.com/content/releasing/copyright-headers.html does seem to imply we shouldn't remove copyright headers. A lot of the patch files already run afoul of this.

**Comment by vmoroz** [comment]
Yes, this doc says: "We should never remove copyright notices from files." explicitly. We must go back to files where we did it by mistake and fix them.  

---
In reply to: [585326036](https://github.com/microsoft/react-native-windows/pull/4090#issuecomment-585326036) [](ancestors = 585326036)

**Pull request closed without merging**